### PR TITLE
Add BigtableWhileMatchResultScannerAdapte

### PR DESCRIPTION
**Pull request by kevinsi4508**

### CODE REVIEW

**Comment by kevinsi4508** [comment]
from pull/423

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestBigtableWhileMatchResultScannerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package com.google.cloud.bigtable.hbase.adapters;
```
Comment (sduskis) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestBigtableWhileMatchResultScannerAdapter.java]: Add the license here.

Comment (kevinsi4508) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestBigtableWhileMatchResultScannerAdapter.java]: done

**Comment by sduskis** [comment]
LGTM

**Pull request merged**